### DISCUSSION TITLE
从12升级到13后，快捷键的显示问题

### DISCUSSION THREAD

**Discussion by euzenlee** [Chaoses-Ib/DirectoryOpus#12] [提问] [answered]
全新安装的13版本，快捷显示方式是有个外框的
![image](https://github.com/user-attachments/assets/c76b773b-ade2-4bb9-87aa-86eae3376acb)

从12升级过来的版本，则是传统的样式，找了一圈也没看到在哪里设置
![image](https://github.com/user-attachments/assets/6f6bf147-36fb-49d9-ac5c-ee8a241bdba7)

**Comment by Chaoses-Ib** [comment]
感觉是主题的问题，你在主题设置里重新应用下默认主题看看？

**Reply by euzenlee** [reply]
应用过默认主题，深色浅色都试过了。也试过全部恢复出厂设置，但结果都不成功。

**Reply by Chaoses-Ib** [reply, marked as answer]
https://github.com/Chaoses-Ib/DirectoryOpus/blob/4944ce3d7d857de88b2144a4f8041367190f2361/浏览/窗口/工具栏/README.md#工具栏升级问题

![image](https://github.com/user-attachments/assets/292cc184-3aaa-427b-a3cd-6f608f594026)
